Doc/library/time.rst

```diff
@@ -126,6 +126,12 @@ The module defines the following functions and data items:
    This is negative if the local DST timezone is east of UTC (as in Western Europe,
    including the UK).  Only use this if ``daylight`` is nonzero.
 
+   .. note::
```

Comment:
Thank you.  I thought that it wasn't DRY when I was changing it but didn't know about moving things around.  I step back and look at the overall layout.